web/src/features/auth/slices/authenticationSlice.test.ts

```diff
@@ -47,6 +47,7 @@ describe('authenticationSlice', () => {
         ...initialState,
         authenticating: false,
         isAuthenticated: true,
+        idir: 'cbrady@idir',
```

Comment:
Maybe use a totally fake idir?

Reply:
It needs to come from the token, would need to generate a token somehow with a fake idir.

Reply:
no - i mean, just don't use your own idir here.
```suggestion
        idir: 'test@idir',
```
Since we're just making it up
```
if (TEST_AUTH || window.Cypress) {
    return 'test@idir'
}
```

Reply:
We'd lose a lot of test coverage doing this since decode roles logic is short circuited when `TEST_AUTH` is set. JWT decoding of keycloak token would not be tested as well as setting and unsetting of roles in other reducer functions.

Reply:
As discussed, mocked out jwt decode calls and removed token/idir strings